src/db/profile.sql

```diff
@@ -0,0 +1,10 @@
+-- Volunteer Profile Schema
+CREATE TABLE IF NOT EXISTS profile (
+    id TEXT PRIMARY KEY,               -- UUID stored as TEXT
+    email TEXT NOT NULL UNIQUE,
+    schemaVersion INTEGER NOT NULL,
+    name TEXT NOT NULL,
+    description TEXT,
+    happenedAt TEXT NOT NULL,
+    insertedAt TEXT NOT NULL
+);
```

Comment:
is `socialLinks` still part of the profile schema? it's included in the current dummy data: https://github.com/torontojs/vms/blob/ebfb81654699a56dfbd4315f1e243fac472f9830/src/data/profiles.json#L10-L12

Reply:
It was not before... But we can reinstate that property.

Reply:
This sounds like an okay solution, yes.